api/resources/get_transaction.py

```diff
@@ -0,0 +1,36 @@
+from flask_restful import Resource
+from flask import request
+from config import app
+import iota
+import json
+
+
+class Get_transaction (Resource):
+    def get(self, tx_hash):
+        if len(tx_hash) != 81:
+            return {
+                'message': 'Type Not Included!'
+            }, 400
+
+        api = iota.Iota(app.config['API_URI'])
```

Comment:
`get_transaction` can be a utils function.

Reply:
Fix in f01cf8f.